index.html

```diff
@@ -216,6 +216,48 @@ <h4 class="tab-caption-skill">Skills</h4>
             </ul>
      </div>
    </section>
```

Comment:
- [ ] **[OPTIONAL]** Congratulations on implementing the suggestion for adjusting the `width` of your contact form **based on the previous reviewer's feedback!** I wanted to offer one **more optional suggestion for you to consider**. It would be great if you could take a look at the `width` of some of your project cards as they appear s**lightly off-center and smaller compared to the [template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122).** Adding this adjustment would further enhance the overall presentation.😉

   ### Your design:
   ![image](https://github.com/ariful04/Myportfolio/assets/107952724/b0d99b3b-c18e-4b1c-9dba-114da410100d)